src/Containers.jsx

```diff
@@ -79,9 +79,43 @@ class Containers extends React.Component {
         }));
     }
 
-    // TODO
     handleContainerCommit(commitMsg) {
+        let cmdStr = "";
+        if (commitMsg.command.trim() === "") {
+            cmdStr = this.state.containerWillCommit.Config ? this.state.containerWillCommit.Config.Cmd.join(" ") : "";
+        } else {
+            cmdStr = commitMsg.command.trim();
+        }
+
+        let commitData = {};
+        commitData.name = this.state.containerWillCommit.ID;
+        commitData.image_name = commitMsg.tag ? commitMsg.imageName + ":" + commitMsg.tag : commitMsg.imageName;
+        commitData.author = commitMsg.author;
+        commitData.message = commitMsg.message;
+        commitData.pause = commitMsg.pause;
+        commitData.format = commitMsg.format;
+
+        commitData.changes = [];
+        let cmdData = "CMD=" + cmdStr;
+        commitData.changes.push(cmdData);
+
+        let onbuildsArr = [];
+        if (commitMsg.setonbuild) {
+            onbuildsArr = utils.getCommitArr(commitMsg.onbuild, "ONBUILD");
+        }
+        commitData.changes.push(...onbuildsArr);
 
+        utils.varlinkCall(utils.PODMAN, "io.podman.Commit", commitData)
+                .then(reply => {
+                    this.props.updateImagesAfterEvent();
+                    this.props.updateContainersAfterEvent();
+                })
+                .catch(ex => {
+                    console.error("Failed to do Commit call:", ex, JSON.stringify(ex));
```

Comment:
If Commit call failed the user should see error notification in the open dialog.
In other words the dialog wondow should not close IMO if:
* Commit varlinkCall failed
* Some of the options are invalid (we can perform some empty string checks there for example)